django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+
+class Comment(models.Model):
+    commented_object = GenericForeignKey()
+    content_type = models.ForeignKey(
+        ContentType,
+        on_delete=models.CASCADE,
+        related_name="comments",
+    )
+    object_id = models.PositiveIntegerField()
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="comments",
+    )
+    content = models.TextField(
+        max_length=2048,
+    )
+    date_created = models.DateTimeField(
+        auto_now_add=True,
+    )
+    date_updated = models.DateTimeField(
+        auto_now=True,
+    )
+    uuid4 = models.UUIDField(
+        default=uuid.uuid4,
```

Comment:
I know this isn't what the Package model does, but when using uuid I like to use https://github.com/valohai/ulid2 rather than just a plain uudi4 for some nicer guarantees